docs/spec/admission.md

```diff
@@ -55,22 +55,22 @@ If and only if the Pod has the `ns.networkservicemesh.io` annotation exists, and
 
 ```yaml
 initContainers:
-       - name: nsc
-         image: ${REPO}/${INITCONTAINER}:${TAG}
-         imagePullPolicy: IfNotPresent
-         env:
-           - name: NS_NETWORKSERVICEMESH_IO
-             value: ${value of annotation}
-         resources:
-           limits:
-             networkservicemesh.io/socket: 1
+   - name: nsm-init-container
```

Comment:
Minor nit: this indent level (3 spaces) is inconsistent with other levels (2 spaces) there and in above examples.